_pages/dev/features/coupons.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Coupons (DRAFT)
+---
+
+## Intro
+
+When a coupon promotion has been created, customers apply coupons that are applicable to the promotion during checkout to get discounts and/or rewards for their orders. 
```

Comment:
Please replace the above sentence with this one: 
"When a coupon promotion has been created, customers apply the applicable coupon to activate the promotion during checkout to get discounts and/or rewards for their orders. "